### DISCUSSION TITLE
Why is there a big price difference between infracost and gcp calculator?

### DISCUSSION THREAD

**Discussion by Alexlofern** [infracost/infracost#2472] [General]
<img width="342" alt="image" src="https://github.com/infracost/infracost/assets/27426979/0fc1fdd9-c2f0-47e9-8494-34e4272a4a0b">
<img width="232" alt="image" src="https://github.com/infracost/infracost/assets/27426979/403274e6-96d0-40c0-8c2a-b7e831fb3dc8">

I'm using 0.10.22 version.

Regards

Alexandre

**Comment by aliscott** [comment]
Moving this to an issue and closing this discussion: https://github.com/infracost/infracost/issues/2473
